File: webapp/HIN_WEB/ClientApp/src/app/model/template.ts
export class Template {
  public Id: number;
  public PatientName: string;
  public AppointmentId: number;
  public Sex: string;
  public Age: string;
  public ChiefCompliant: string;
  public Date: Date;
  public DateOfAccident: Date;
  public HistoryOfIllness: string;
  public UpdatedOn: Date;
  public UpdatedBy: string;
  public CreatedOn: Date;
  public CreatedBy: string;
  public TemplateId: number;
  public DraftHtml: string;
  public Status: TemplateStatus;
}
export enum TemplateStatus {
  Draft = 0,
  Sent = 1
}

export class TemplateMailRequest {
  public MailId: string;
  public HtmlBody: string;
  public Appointment: string;
}

export class SearchOnedrive {
  public Location: string;
  public PracticeName: string;
  public PracticeDate: Date;
  public PatientName: string;
}

File: webapp/HIN_WEB/Models/Template.cs
using System;

namespace HIN_WEB.Models
{
    public class Template
    {
        public int Id { get; set; }
        public string PatientName { get; set; }
        public int? AppointmentId { get; set; }
        public string Sex { get; set; }
        public string Age { get; set; }
        public string ChiefCompliant { get; set; }
        public DateTime? Date { get; set; }
        public DateTime? DateOfAccident { get; set; }
        public string HistoryOfIllness { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public int? TemplateId { get; set; }
        public string DraftHtml { get; set; }
        public int? Status { get; set; }
    }
}
